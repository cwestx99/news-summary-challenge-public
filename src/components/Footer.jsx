const Footer = () => {

    return (
        <>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
            <footer className="footer">
                <p>Christopher West Â© - 2022</p>

                <ul>
                    <a class="fa fa-facebook" />
                    <a class="fa fa-twitter" />
                    <a class="fa fa-instagram" />
                </ul>

            </footer>
        </>
    );
};

export default Footer