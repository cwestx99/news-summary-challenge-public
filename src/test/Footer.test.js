import { render } from '@testing-library/react';
import Footer from '../Components/Footer';

test(`Footer matches snapshot`, () => {
    const mockFooter = render(<Footer />);
    expect(mockFooter).toMatchSnapshot();
});