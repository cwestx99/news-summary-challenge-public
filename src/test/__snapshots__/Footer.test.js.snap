// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <link
        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
        rel="stylesheet"
      />
      <footer
        class="footer"
      >
        <p>
          Christopher West © - 2022
        </p>
        <ul>
          <a
            class="fa fa-facebook"
          />
          <a
            class="fa fa-twitter"
          />
          <a
            class="fa fa-instagram"
          />
        </ul>
      </footer>
    </div>
  </body>,
  "container": <div>
    <link
      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
      rel="stylesheet"
    />
    <footer
      class="footer"
    >
      <p>
        Christopher West © - 2022
      </p>
      <ul>
        <a
          class="fa fa-facebook"
        />
        <a
          class="fa fa-twitter"
        />
        <a
          class="fa fa-instagram"
        />
      </ul>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
